elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,252 +3,519 @@
-      border-top-color: transparent;
-      border-right: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-var(ui-accent)};
+      @include browser-query(ie11) {
+        color: $pfe-color--black;
+        border-top-color: transparent;
+        border-right: pfe-fetch(ui--border-width--active) pfe-var(ui--border-style) pfe-var(ui-accent);
+        border-bottom: 0;
+      }
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"]) {
+      border-left: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
+      text-align: left !important;
+
+      @include browser-query(ie11) {
+        border-left: pfe-fetch(ui--border-width) pfe-fetch(ui--border-style) pfe-fetch(surface--border);
+      }
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="true"]) {
+      border-right: #{pfe-local(BorderWidth)} #{pfe-var(ui--border-style)} transparent;
+      border-left: #{pfe-local(BorderWidth)} #{pfe-var(ui--border-style)} #{pfe-local(highlight)};
+      padding-left: calc(#{pfe-var(container-padding)} - 2px);
+
+      @include browser-query(ie11) {
+        border-right: 0;
+        border-left: pfe-local(BorderWidth) pfe-fetch(ui--border-style) pfe-local(highlight);
+        padding-left: 10px;
+        border-bottom: 0 !important;
+      }
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]) {
+      border-right: #{pfe-local(BorderWidth)} #{pfe-var(ui--border-style)} transparent;
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]:hover) {
+      border-right: #{pfe-local(BorderWidth)} #{pfe-var(ui--border-style)} transparent;
+      border-Bottom: 0;
+      border-left: #{pfe-local(BorderWidth)} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor--hover)};
+      padding-left: calc(#{pfe-var(container-padding)} - 2px);
+
+      @include browser-query(ie11) {
+        border-right: 0;
+        border-bottom: 0;
+        padding-left: 10px;
+        border-left: pfe-local(BorderWidth) pfe-fetch(ui--border-style) pfe-local(BorderColor--hover);
+      }
     }
-  }
-}
 
-:host([pfe-variant="earth"]) {
-  @include browser-query(ie11) {
-    background-color: #{pfe-var(surface--lighter)};
-    color: #{pfe-var(text)};
   }
+  
 }
 
 /// ===========================================================================
 /// WIND TAB VARIANTS / DEFAULT
 /// ===========================================================================
 
-@each $theme in $THEMES { // dark, saturated, and light
-  :host([aria-selected="true"][on="#{$theme}"]),
-  :host([on="#{$theme}"]:hover),
-  :host([pfe-variant="wind"][aria-selected="true"][on="#{$theme}"]),
-  :host([pfe-variant="wind"][on="#{$theme}"]:hover) {
-    @if $theme != "light" {
-      --pfe-tabs--Color: #{pfe-var(text--on-#{$theme})};
+:host([pfe-variant="wind"]) .pfe-tab {
+  display: inline-block !important;
+}
+
+:host {
+  text-align: left;
```

Comment:
This is a global host style hanging out in the wind variant section; might be good to migrate this to the host at the top of the file

Reply:
migrated up to the top